src/main/java/me/desht/dhutils/MiscUtil.java

```diff
@@ -197,4 +201,20 @@ else if (regexMatcher.group(2) != null) {
     public static boolean looksLikeUUID(String s) {
         return s.length() == 36 && s.charAt(8) == '-' && s.charAt(13) == '-' && s.charAt(18) == '-' && s.charAt(23) == '-';
     }
+
+    /**
+     * Truncates a decimal to a given number of places.
+     *
+     * @param x the decimal to truncate
+     * @param places the number of places to truncate
+     * @return x truncated to places places
+     */
+    @Nonnull
+    public static double truncateDecimal(double x, int places) {
+        if (x > 0) {
+            return new BigDecimal(String.valueOf(x)).setScale(places, RoundingMode.FLOOR).doubleValue();
+        } else {
+            return new BigDecimal(String.valueOf(x)).setScale(places, RoundingMode.CEILING).doubleValue();
+        }
+    }
```

Comment:
`DecimalFormat` seems much more useful for this here, look at Slimefun's NumberUtils, you can copy that method probably